### ISSUE TITLE
Feature Request: At board level, hide label with text on minicard

### ISSUE THREAD

**Issue by javen9881** [wekan/wekan#2561]
Do we have this function like Trello?
![image](https://user-images.githubusercontent.com/46272743/61763656-b455bd00-ae08-11e9-96ed-3afcc8cb519b.png)

**Comment by xet7** [comment]
@javen9881 

In Wekan, minicard label text is in color tooltip, not in actual color. You see text when you hover over that color on minicard.

**Comment by xet7** [comment]
@javen9881 

I tried adding that feature:

![label-texts](https://user-images.githubusercontent.com/15545/61803624-31018f00-ae3b-11e9-80ff-319521c24b87.png)

Although that card has too many labels. If there is less labels, labels are aligned to right. On that screenshot it seems labels are aligned to left. Anyway, it seems it's better to have text in labels at minicards.

**Comment by javen9881** [comment]
thanks for your quick response @xet7 
have a nice day!

**Comment by LyR33x** [comment]
Thank you for that!
I would prefer if I can turn this text off again in settings. I'm using a lot of labels with long text. The card getting a extrem width if I assign some labels to the card and the complete template gets confusing.

**Comment by hever** [comment]
Yes. As mentioned at #2296 there should be some switch at board level. Somewhere it is good, but somewhere it is so disturbing.

**Comment by LyR33x** [comment]
Ok nice! 
Translation not working so far (always english) and its not on Board-Level. If I switch it, all boards switching and not only the board I clicking the switch. If you want this like that, dann the switch is in the wrong place (should be in user-settings then) otherwise you need to fix these bug.

**Comment by xet7** [comment]
I just today did figure out how to hide minicard label text, looking at other activities hide implementation that happened to be per user. It's not at board level yet, maybe I'll figure it out later.

I just today added this feature, so it's not yet translated to all languages by Wekan translators from all around the world https://transifex.com/wekan/wekan . Other translations will be added later.

I will also later try to figure out how to limit card width, so that labels would be at multiple lines.

Well, is something a bug or a feature, it depends on each persons opinions. I try to think what option makes some sense.
